server/mediaserver.go

```diff
@@ -962,6 +963,51 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getPlalistsFromStore(ctx context.Context, sess drivers.OSSession, manifests []string) (map[string][]int, []string, time.Time, error) {
```

Comment:
I think a short comment describing what this function is supposed to return would be helpful.

Personally, took me a bit to understand that it returns:

- A map of manifestID -> a list of indices pointing to JSON files in the returned list of JSON files
- A list of JSON files for all manifestIDs provided
- The latest playlist time

Reply:
Fixed